src/js/techreport/table.js

```diff
@@ -194,7 +194,7 @@ function updateTable(id, config, appConfig, apps, data) {
       }
 
       if(column.styling) {
-        wrapper.classList.add('monthchange', column.styling.direction, column.styling.color);
+        wrapper.className = `monthchange ${styling.color} ${styling.direction}`;
```

Comment:
Doesn't this need to be:

```suggestion
        wrapper.className = `monthchange ${column.styling.color} ${column.styling.direction}`;
```

Reply:
Fixed this in the merge with `main` from #1068